metabrainz/static/css/main.less

```diff
@@ -38,6 +38,12 @@ body {
 img { .img-responsive(); }
 img.pull-right { padding-left: 15px; }
 
+
+a.support-link {
+  color: lighten(@green, 30%) !important;
+}
```

Comment:
Let's please try a red background (same as the under construction red bar) with a bold white text.